### ISSUE TITLE
Add styling using tailwind CSS

### ISSUE THREAD

**Issue by krishp058** [ScriptedRain/React-chat-app#6]
@bakhosRachwan Would you mind helping us complete the CSS for the chat page and becoming a part of our head development team. You are welcome to become our lead contributor. 

It would benefit and mean a lot if you join up with us, since you have great experience in Firebase, tailwind, react.js.

Looking forward to working with you

**Comment by krishp058** [comment]
Please continue styling from the 'Dev' branch since that is where we are styling.

**Comment by bakhosRachwan** [comment]
Yes I would like to help, but i just have to note that I live in GMT+3 Timezone in case I'm not always online when you are.

Now for the design do you have anything in mind. So that i know how you want the layout and colors and all that.
Or do you want the same design from the video?

**Comment by ScriptedRain** [comment]
ok welcome to the team! 

The landing page should be as it is right now, but you can play around with the chat page but make it dark theme and with the accent colours which are blue and purple.

**Comment by ScriptedRain** [comment]
Also, the program name is Vybe in case you needed to know, thanks. 👍

**Comment by krishp058** [comment]
Hi there, be aware that this project was just a way for us to learn how to use Firebase etc. But we are planning to create a larger project near the end of this week which will be more capable and have better functionality such as Private DMs, as well as group messages, texting with pictures. You are welcome to join in on that if you want to.

**Comment by bakhosRachwan** [comment]
Yes count me in if you want on that bigger project. however i suggest we chat over discord or slack when you want to do the other project. because it is better to decide on the design of the app first and then divide it into components. that makes managing the project easier.

**Comment by krishp058** [comment]
Ok sure.

**Comment by mars-s** [comment]
@bakhosRachwan ok i made a channel for us developers on slack send me the email here so i can add you

**Comment by bakhosRachwan** [comment]
@ScriptedRain  it's bakhos.rach@gmail.com
